src/components/Molecules/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,24 @@
+import css from './Navbar.module.css';
+import links from './links.json';
+
+import { NavigationLink } from 'components/Atoms/NavigationLink/NavigationLink.jsx';
+
+const Navbar = () => {
+  return (
+    <nav className={css.navigation}>
+      <ul className={css.list}>
+        {links.map((link, index) => (
+          <li key={index}>
+            <NavigationLink
+              path={link.path}
+              title={link.title}
+              icon={link.icon}
+            />
+          </li>
```

Comment:
Iteration outside rendering ❌

Reply:
Some tips how to resolve that? I have no idea. I made a google research and it seems using map() inside rendering is normal practice.